tests/snuba/api/endpoints/test_organization_tagkey_values.py

```diff
@@ -195,15 +195,11 @@ def setUp(self):
         )
 
     def test_status(self):
-        self.run_test("transaction.status", expected=[("unknown_error", 1), ("ok", 1)])
+        self.run_test("transaction.status", expected=[("unknown", 1), ("ok", 1)])
```

Comment:
Will customers start seeing 'unknown' instead of 'unknown_error' now?

Reply:
I assume so, yes, although I do not fully understand how that endpoint ties into the product. 

To my knowledge, `unknown_error` has been deprecated for a while and does not adhere to the opentelemetry spec. If we want to clean this up, now is the time. It should have been changed a long while ago already.

Reply:
not necessarily with this PR but with the corresponding bump of relay in snuba

Reply:
@untitaker Maybe you can shed some light on this. How did this change already if the test suite is still running an unpatched snuba?

Reply:
Ah, of course the test changed. Since we normalize the event before storing, it now receives the normalized status name (which is "unknown" rather than "unknown_error"). This is simply inherited. 

Snuba should map all of these to the same constant internally, so I'm assuming that the counts are not affected.

Reply:
Seems like we're using the mapping exposed by relay in `src/sentry/tagstore/snuba/backend.py` and as such this actually impacts the UI.

That sort of implies we're transmitting the integer values over the wire, which I am not exactly thrilled to see.

Reply:
The end users only ever see the string names in the UI. When sentry talks to snuba though, we use send the integer values as that is what snuba has stored in clickhouse.